Makefile

```diff
@@ -55,8 +54,15 @@ cdx_toolkit:
 	@echo
 
 download_collinfo:
-	@echo downloading collinfo.json so we can find out the crawl name
-	wget https://index.commoncrawl.org/collinfo.json
+	@echo "downloading collinfo.json so we can find out the crawl name"
+	@if command -v wget > /dev/null 2>&1; then \
+		wget https://index.commoncrawl.org/collinfo.json; \
+	elif command -v curl > /dev/null 2>&1; then \
+		curl -O https://index.commoncrawl.org/collinfo.json; \
+	else \
+		echo "Error: neither wget nor curl is installed."; \
+		exit 1; \
+	fi
```

Comment:
```suggestion
	curl -O https://index.commoncrawl.org/collinfo.json;
```

Reply:
Windows also has curl pre-installed from Win 10+, but if-then-else structure to give an error was nice.